utils/src/ooinstall/openshift_ansible.py

```diff
@@ -11,30 +11,46 @@
 
 CFG = None
 
+ROLES_TO_GROUPS_MAP = {
+    'master': 'masters',
+    'node': 'nodes',
+    'storage': 'nfs'
+}
+
+VARIABLES_MAP = {
```

Comment:
Having a single mapping for variables seems a bit odd. I would  think there should be at minimum:
- mapping dict for deployment level vars
- mapping dict for role level vars
- mapping dict for host level vars

And potentially have sets of mappings that could appear at more than one level as well.